include/service/entities/content/article_service_v2.js

```diff
@@ -140,12 +140,6 @@ module.exports = function(pb) {
         dto.url = BaseObjectService.sanitize(dto.url);
         dto.publish_date = BaseObjectService.getDate(dto.publish_date);
         
-        if (util.isArray(dto.meta_keywords)) {
-            for (var i = 0; i < dto.meta_keywords.length; i++) {
-                dto.meta_keywords[i] = BaseObjectService.getDate(dto.meta_keywords[i]);  
```

Comment:
This was attempting to parse meta_keyword values as dates, which was obviously causing problems.